### ISSUE TITLE
Password must be plaintext

### ISSUE THREAD

**Issue by f4hy** [fagga/transmission-remote-cli#173]
Is this a limitation of transmission rpc itself? It seems like there is no secure way to use this without having my username and password either stored in plaintext, or written out in plaintext at the command line. This seems dangerous to me.

Am I missing something or is there a way to login without ever writing the password in plaintext anywhere.

**Comment by fagga** [comment]
Yes, this isn't exactly state-of-the-art security. Although storing secrets in
files is fairly secure with the correct permissions IMO.

If you need better security, use SSH or SSL.
